src/commands/app/add/extension.js

```diff
@@ -59,7 +58,8 @@ class AddExtensionCommand extends BaseCommand {
 
   async selectImplementations (flags, config) {
     const alreadyImplemented = config.implements
-    const availableChoices = implPromptChoices
+    const consoleCLI = await this.getLibConsoleCLI()
+    const availableChoices = await getImplPromptChoices(consoleCLI, config.aio)
```

Comment:
I thought we decided that add extension and init should let the user choose extension points to implement from an hardcoded list, for two reasons:

- there is no added benefit of adding an extension from a dynamic list for now because we anyway maintain the list in the CLI
- dynamic list requires an extra request, slows down + more chance for bugs

Reply:
cc @meryllblanchet @purplecabbage @sarahxxu